app/src/androidTest/java/com/android/unio/components/event/EventDetailsTest.kt

```diff
@@ -304,13 +335,33 @@ class EventDetailsTest : TearDown() {
   @Test
   fun testGalleryDoesNotDisplayWhenNoPictures() {
     setEventScreen(events[2])
-    composeTestRule
-        .onNodeWithTag(EventDetailsTestTags.EVENT_DETAILS_PAGER)
-        .assertDisplayComponentInScroll()
-    composeTestRule.onNodeWithTag(EventDetailsTestTags.EVENT_DETAILS_PAGER).performScrollToIndex(1)
+    goToGallery()
     composeTestRule.onNodeWithTag(EventDetailsTestTags.GALLERY_GRID).assertIsNotDisplayed()
     composeTestRule
         .onNodeWithTag(EventDetailsTestTags.EVENT_NO_PICTURES_TEXT)
         .assertDisplayComponentInScroll()
   }
+
+  @Test
+  fun testFullSizePictureOnClick() {
+    setEventScreen(events[0])
```

Comment:
Good test addition, even though I think this is a test for a feature implemented in a different PR